data-products/example_prison_data_product/application/transform.py

```diff
@@ -0,0 +1,104 @@
+import pandas as pd
+import logging
+import boto3
+import yaml
+import os
+from pathlib import Path
+
+
+logging.basicConfig()
+s3_client = boto3.client("s3")
+
+
+def get_data(bucket: str, key: str) -> pd.DataFrame:
+    response = s3_client.get_object(Bucket=bucket, Key=key)
+
+    status = response.get("ResponseMetadata", {}).get("HTTPStatusCode")
+
+    if status == 200:
+        logging.info("Data has been collected from {}/{}".format(bucket, key))
+        return pd.read_csv(response.get("Body"))
+    else:
+        logging.error(
+            "Bucket {} or file {} does not exist".format(bucket, key))
+        raise
+
+
+def get_tables(bucket, key, source_data):
```

Comment:
```suggestion
def get_tables(bucket, key, source_data) -> Dict[str: str]:
```
is this correct? I am suggesting adding typing to the functions